src/components/MultipleChoice/MultipleChoice.stories.tsx

```
@@ -0,0 +1,39 @@
+      <div className="space-y-6">
+        <h2 className="text-3xl font-bold text-text-900 border-b border-border-100 pb-2">
+          Modo Interativo
+        </h2>
+
+        <div className="grid grid-cols-1 lg:grid-cols-2 gap-8">
+          <div className="space-y-4">
+            <h3 className="text-xl font-semibold text-text-800">
+              Layout Padrão
+            </h3>
+            
+          </div>
```

ESLint

[error] Delete `··⏎··········`

(prettier/prettier)